telegram/ext/_extbot.py

```diff
@@ -202,7 +202,7 @@ def insert_callback_data(self, update: Update) -> None:
             *In place*, i.e. the passed :class:`telegram.Message` will be changed!
 
         Args:
-            update (:class`telegram.Update`): The update.
+            update (:class:`telegram.Update`): The update.
```

Comment:
ah this was in the referencing pr so I'd left it

Reply:
ah yes, didn't think of that :D shouldn't hurt, though …